config/install/field.storage.paragraph.lgms_width_of_section.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.lgms_width_of_section
+field_name: lgms_width_of_section
+entity_type: paragraph
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: full-width
+      label: 'Full width of screen'
+    -
+      value: full-width-contained-content
+      label: 'Full width, with content contained'
```

Comment:
Not sure I understand this one. Is it wider than the content but not fullscreen? 
Maybe we could call it "Breakout" ?

Reply:
Great idea!